### DISCUSSION TITLE
Difference between user_id and session_id

### DISCUSSION THREAD

**Discussion by ACPK** [plausible/analytics#991] [Self-Hosted Support] [answered]
When reviewing ```plausible_events_db.sessions```, what is the difference between user_id and session_id?

Additionally, what will cause sessions to have an already used session_id (114 rows with 110 distinct session_ids)?

As of today, our test database has:

1. 114 rows sessions
2. 110 rows with a distinct session_id
3. 99 rows with a distinct user_id

I noticed that the "unique visitors" number is equal to the 99 rather than than 110.

**Comment by ACPK** [comment, marked as answer]
Any ideas?

**Reply by metmarkosaric** [reply]
one unique visitor can come back to a site on the same day which means that the unique visitor then has multiple sessions. it's not something that we display in the dashboard though
